system-int-test/src/cmr/system_int_test/utils/index_util.clj

```diff
@@ -68,6 +69,18 @@
                     :throw-exceptions false})]
     (is (= 200 (:status response)) (:body response))))
 
+(defn doc-present?
+  "If doc is present return true, otherwise return false"
+  [index-name type-name doc-id]
+  (let [response (client/get
+                  (format (str (url/elastic-root) "/%s/%s/_search?_id=%s") index-name type-name doc-id)
```

Comment:
Since you are using `format` you may as well get rid of `str` `(format "%s/%s/%s_search?_id=%s" (url/elastic-root) index-name type-name doc-id)`